src/main/java/teammates/common/util/Const.java

```diff
@@ -937,7 +937,7 @@ private Const() {
         public static final String STUDENTS = "/students";
         public static final String STUDENT = "/student";
         public static final String SESSIONS_ADMIN = "/sessions/admin";
-        public static final String SESSIONS_STATS = "/sessions/stats";
+        public static final String SESSIONS_STATS = "/session/stats";
```

Comment:
Change the variable name to `SESSION_STATS` to reflect the new URI.

Reply:
And move it down to after `/session` for neater organization.

Reply:
Do we need to move it down to after `SESSION_REMIND_RESULT` (line 944) for alphabetical ordering?

Reply:
Sure.